package.json

```diff
@@ -21,10 +21,10 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.20",
-    "@docusaurus/module-type-aliases": "^2.0.0-beta.20",
-    "@docusaurus/preset-classic": "^2.0.0-beta.20",
-    "@docusaurus/theme-common": "^2.0.0-beta.20",
+    "@docusaurus/core": "^0.0.0-5035",
```

Comment:
is this normal?

Reply:
See description.

Reply:
on mobile here, thanks for pointing out, missed that :)

Reply:
As for the formatting, that's just the pre-commit hook. Seems like they were skipped before.